tests/php/validation/test-class-amp-validation-manager.php

```diff
@@ -1747,7 +1747,9 @@ public function test_enqueue_block_validation() {
 			'wp-element',
 			'wp-hooks',
 			'wp-i18n',
+			'wp-nux',
 			'wp-polyfill',
+			'wp-url',
```

Comment:
These are probably from the dependencies in [amp-preview.js](https://github.com/ampproject/amp-wp/pull/3323/files#diff-66f94e980255259f2c596579f2853515R14-R16).